lib/mini-lightbox.js

```diff
@@ -1,6 +1,45 @@
 // MiniLightbox
 // </> with <3 by Ionică Bizău
-(function (root) {
+(function () {
```

Comment:
Ahh, okay. You want to remove the [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE), right?
I wouldn't advise for that to be honest, since that would break the usage for everyone that installed the module to date.

It's one of the annoying things in browser JS development at the moment, that you have to provide a lot of boilerplate code to satisfy the needs of lots of different module loaders. This one is surely extensive. But that means less administrative effort in the future.